/etc/gdm3/Xsession: Beginning session setup...
localuser:hacker being added to access control list
Failed to connect to the VirtualBox kernel service
Failed to connect to the VirtualBox kernel service
Failed to connect to the VirtualBox kernel service
Failed to connect to the VirtualBox kernel service
gnome-keyring-daemon: insufficient process capabilities, unsecure memory might get used
gnome-keyring-daemon: insufficient process capabilities, unsecure memory might get used
gnome-keyring-daemon: insufficient process capabilities, unsecure memory might get used
gnome-keyring-daemon: insufficient process capabilities, unsecure memory might get used
GNOME_KEYRING_CONTROL=/home/hacker/.cache/keyring-4rtmrz
GNOME_KEYRING_CONTROL=/home/hacker/.cache/keyring-4rtmrz
SSH_AUTH_SOCK=/home/hacker/.cache/keyring-4rtmrz/ssh
GNOME_KEYRING_CONTROL=/home/hacker/.cache/keyring-4rtmrz
SSH_AUTH_SOCK=/home/hacker/.cache/keyring-4rtmrz/ssh
GPG_AGENT_INFO=/home/hacker/.cache/keyring-4rtmrz/gpg:0:1
GNOME_KEYRING_CONTROL=/home/hacker/.cache/keyring-4rtmrz
SSH_AUTH_SOCK=/home/hacker/.cache/keyring-4rtmrz/ssh
GPG_AGENT_INFO=/home/hacker/.cache/keyring-4rtmrz/gpg:0:1

(gnome-settings-daemon:3099): power-plugin-WARNING **: Failed set DPMS mode: Display is not DPMS capable

(gnome-settings-daemon:3099): color-plugin-WARNING **: failed to get edid: unable to get EDID for output

(gnome-settings-daemon:3099): color-plugin-WARNING **: unable to get EDID for xrandr-default: unable to get EDID for output

(gnome-settings-daemon:3099): color-plugin-WARNING **: failed to reset xrandr-default gamma tables: gamma size is zero

(gnome-panel:3141): Gtk-CRITICAL **: gtk_accelerator_parse_with_keycode: assertion `accelerator != NULL' failed

** (gnome-panel:3141): WARNING **: Unable to parse mouse modifier '(null)'


(gnome-settings-daemon:3099): updates-plugin-WARNING **: state unknown: 0

(gnome-settings-daemon:3099): PackageKit-WARNING **: failed to set proxy: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 654, in _message_cb
    (candidate_method, parent_method) = _method_lookup(self, method_name, interface_name)
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 246, in _method_lookup
    raise UnknownMethodException('%s is not a valid method of interface %s' % (method_name, dbus_interface))
UnknownMethodException: org.freedesktop.DBus.Error.UnknownMethod: Unknown method: SetProxy is not a valid method of interface org.freedesktop.PackageKit


(gnome-settings-daemon:3099): updates-plugin-WARNING **: failed to set proxies: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 654, in _message_cb
    (candidate_method, parent_method) = _method_lookup(self, method_name, interface_name)
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 246, in _method_lookup
    raise UnknownMethodException('%s is not a valid method of interface %s' % (method_name, dbus_interface))
UnknownMethodException: org.freedesktop.DBus.Error.UnknownMethod: Unknown method: SetProxy is not a valid method of interface org.freedesktop.PackageKit


(gnome-settings-daemon:3099): PackageKit-WARNING **: failed to set root: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 654, in _message_cb
    (candidate_method, parent_method) = _method_lookup(self, method_name, interface_name)
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 246, in _method_lookup
    raise UnknownMethodException('%s is not a valid method of interface %s' % (method_name, dbus_interface))
UnknownMethodException: org.freedesktop.DBus.Error.UnknownMethod: Unknown method: SetRoot is not a valid method of interface org.freedesktop.PackageKit


(gnome-settings-daemon:3099): updates-plugin-WARNING **: failed to set install root: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 654, in _message_cb
    (candidate_method, parent_method) = _method_lookup(self, method_name, interface_name)
  File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 246, in _method_lookup
    raise UnknownMethodException('%s is not a valid method of interface %s' % (method_name, dbus_interface))
UnknownMethodException: org.freedesktop.DBus.Error.UnknownMethod: Unknown method: SetRoot is not a valid method of interface org.freedesktop.PackageKit

Initializing tracker-store...
Tracker-Message: Setting up monitor for changes to config file:'/home/hacker/.config/tracker/tracker-store.cfg'
Initializing tracker-miner-fs...
Tracker-Message: Setting up monitor for changes to config file:'/home/hacker/.config/tracker/tracker-store.cfg'
Starting log:
  File:'/home/hacker/.local/share/tracker/tracker-store.log'
Tracker-Message: Setting up monitor for changes to config file:'/home/hacker/.config/tracker/tracker-miner-fs.cfg'
Starting log:
  File:'/home/hacker/.local/share/tracker/tracker-miner-fs.log'
vmware-user: could not open /proc/fs/vmblock/dev
Initializing nautilus-gdu extension
** Message: applet now removed from the notification area

(gnome-panel:3141): Gtk-CRITICAL **: gtk_accelerator_parse_with_keycode: assertion `accelerator != NULL' failed

** (gnome-panel:3141): WARNING **: Unable to parse mouse modifier '(null)'

[1397597439,000,xklavier.c:xkl_engine_start_listen/] 	The backend does not require manual layout management - but it is provided by the application
[1397597439,000,gkbd-status.c:gkbd_status_set_current_page_for_group/] 	Page for group 0 is not ready
[1397597439,000,gkbd-status.c:gkbd_status_set_current_page_for_group/] 	Page for group 0 is not ready
[1397597439,000,gkbd-status.c:gkbd_status_set_current_page_for_group/] 	Page for group 0 is not ready
=== Updating for primary Configuration ===
  Clone: false
  Output: Unknown attached to default
     status: on
     width: 1440
     height: 900
     rate: 60
     primary: true
     position: 0 0
Unable to open desktop file sudo-1.desktop for panel launcher
Unable to open desktop file /usr/share/applications/zaproxy.desktop for panel launcher: No such file or directory

(gnome-panel:3141): GLib-GObject-WARNING **: /build/buildd-glib2.0_2.33.12+really2.32.4-5-i386-eISom6/glib2.0-2.33.12+really2.32.4/./gobject/gsignal.c:2459: signal `size_request' is invalid for instance `0x95fc228'
CachingBackend: Loading instances from cache
CachingBackend: Loading <Notes1>
No global tempdir found, creating new one.
Temp directory /tmp/screenlets created.
Creating new entry for NotesScreenlet in /tmp/screenlets/screenlets.hacker.running
Loading instances in: /home/hacker/.config/Screenlets/Notes/default/
File: Notes1.ini

(gnome-settings-daemon:3099): color-plugin-WARNING **: unable to get EDID for xrandr-default: unable to get EDID for output

(gnome-settings-daemon:3099): color-plugin-WARNING **: failed to reset xrandr-default gamma tables: gamma size is zero

(gnome-settings-daemon:3099): color-plugin-WARNING **: unable to get EDID for xrandr-default: unable to get EDID for output

(gnome-settings-daemon:3099): color-plugin-WARNING **: failed to reset xrandr-default gamma tables: gamma size is zero

(gnome-settings-daemon:3099): color-plugin-WARNING **: unable to get EDID for xrandr-default: unable to get EDID for output

(gnome-settings-daemon:3099): color-plugin-WARNING **: failed to reset xrandr-default gamma tables: gamma size is zero
/home/hacker/.config/Screenlets/Notes/default/Notes1
LOAD NEW THEME: default
FOUND: /usr/share/screenlets/Notes/themes/default
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
WARNING - add_default_menuitems and add_menuitems should be set in on_init ,menu values will be displayed incorrectly
Set options in NotesScreenlet
LOAD NEW THEME: green
FOUND: /usr/share/screenlets/Notes/themes/green
Screenlet has been initialized.
Restored instances from session 'default' ...
** Message: applet now embedded in the notification area
gnome-session[3039]: Gdk-WARNING: gnome-session: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.


(nautilus:3206): GVFS-RemoteVolumeMonitor-WARNING **: Owner :1.11 of volume monitor org.gtk.Private.GduVolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(gnome-fallback-mount-helper:3208): GVFS-RemoteVolumeMonitor-WARNING **: Owner :1.11 of volume monitor org.gtk.Private.GduVolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(nautilus:3206): GVFS-RemoteVolumeMonitor-WARNING **: Owner :1.12 of volume monitor org.gtk.Private.AfcVolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(gnome-fallback-mount-helper:3208): GVFS-RemoteVolumeMonitor-WARNING **: Owner :1.12 of volume monitor org.gtk.Private.AfcVolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(nautilus:3206): GVFS-RemoteVolumeMonitor-WARNING **: Owner :1.13 of volume monitor org.gtk.Private.GPhoto2VolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(gnome-fallback-mount-helper:3208): GVFS-RemoteVolumeMonitor-WARNING **: Owner :1.13 of volume monitor org.gtk.Private.GPhoto2VolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(update-notifier:3219): GConf-WARNING **: Got Disconnected from DBus.


(nm-applet:3194): GConf-WARNING **: Got Disconnected from DBus.

g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.

Received signal:15->'Terminated'

(gnome-settings-daemon:3099): Gdk-WARNING **: gnome-settings-daemon: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Window manager warning: Fatal IO error 11 (Resource temporarily unavailable) on display ':0'.

(gnome-panel:3141): Gdk-WARNING **: gnome-panel: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.


(gdu-notification-daemon:3197): Gdk-WARNING **: gdu-notification-daemon: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

gopenvpn: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.

Received signal:15->'Terminated'Tracker-Message: Closed sqlite3 database:'/home/hacker/.cache/tracker/meta.db'

OK

update-notifier: Fatal IO error 11 (Resource temporarily unavailable) on X server :0.

(tracker-miner-fs:3199): GVFS-RemoteVolumeMonitor-WARNING **: Owner :1.11 of volume monitor org.gtk.Private.GduVolumeMonitor disconnected from the bus; removing drives/volumes/mounts

(tracker-miner-fs:3199): GLib-GIO-CRITICAL **: Error while sending AddMatch() message: The connection is closed

(tracker-miner-fs:3199): GLib-GIO-CRITICAL **: Error while sending AddMatch() message: The connection is closed

(tracker-miner-fs:3199): GLib-GIO-CRITICAL **: Error while sending AddMatch() message: The connection is closed

OK


(gksu:3280): GLib-CRITICAL **: g_str_has_prefix: assertion `str != NULL' failed
